src/lib.rs

```diff
@@ -1121,6 +1121,15 @@ impl Element {
         .await
     }
 
+    /// Find and click an `option` child element by its index.
+    ///
+    /// Index starts from 0.
```

Comment:
This is looking great now! I think the one thing I'd add is some more extensive documentation of this method. In particular, I think it should mention that it expects there to only be one set of `option`s, and that it uses the CSS selector `nth-of-type`.

Reply:
> I think it should mention that it expects there to only be one set of options

Didn't clearly understand the idea so I basically copied that statement :)

Reply:
Try this:
```rust
/// Find and click an `<option>` child element by its index.
///
/// This method clicks the first `<option>` element that is an `index`th child
/// (`option:nth-of-type(index+1)`). This will be the `index`th `<option>`
/// element if the current element is a `<select>`. If you use this method on
/// an `Element` that is _not_ a `<select>` (such as on a full `<form>`), it
/// may not do what you expect if there are multiple `<select>` elements
/// in the form, or if it there are stray `<option>` in the form.
///
/// The indexing in this method is 0-based.
```

Reply:
I've decided to place there a straightforward example how to obtain text from `<select>` when it will be changed (in a way you've pointed me out yesterday) by `prop` since nor `html(true)` nor `text()` is suitable for this as I understand.